orchestrator/lambdas/repo_queue/repo_queue/repo_queue.py

```diff
@@ -27,19 +28,26 @@ def run(event: dict[str, Any] = None, context: LambdaContext = None, services_fi
         data = json.loads(item["body"])
         plugins = data.get("plugins")
         default_branch_only = data.get("default_branch_only", False)
-        repos = query(
-            data["service"],
-            data["org"],
-            services.get(data["service"]),
-            data["page"],
-            default_branch_only,
-            plugins,
-            full_services_dict["external_orgs"],
-            data.get("batch_id"),
-            artemis_api_key,
-            data.get("redundant_scan_query"),
-            data.get("repo"),
-        )
+        repos = []
+
+        try:
+            repos = query(
+                data["service"],
+                data["org"],
+                services.get(data["service"]),
+                data["page"],
+                default_branch_only,
+                plugins,
+                full_services_dict["external_orgs"],
+                data.get("batch_id"),
+                artemis_api_key,
+                data.get("redundant_scan_query"),
+                data.get("repo"),
+            )
+        except HTTPError:
+            log.warning("Unable to Process this organization. Sending task to dead-letter queue")
```

Comment:
We should include some context with this log message (at least service, org, repo, error message) so that we can debug issues in the future.

Reply:
Those keys are already added to the logger in a previous step: 
https://github.com/WarnerMedia/artemis/blob/main/orchestrator/lambdas/repo_queue/repo_queue/repo_queue.py#L74

Any logs statements that come after that step will have the additional keys

Reply:
Oh nice, didn't see that.

I don't see `repo` in there -- is that a useful field to include?

Reply:
The `repo` key is not added in the top-level logger because the `repo` field changes a lot and could give false logs. For example a log could have a `repo` field but its not linked to the named repo

But it is included in the child loggers. here's a spot where the repo field is added in: https://github.com/WarnerMedia/artemis/blob/8009f8b85503939c4aaa95001cdcc8134333575c/orchestrator/lambdas/layers/heimdall_repos/heimdall_repos/github_utils.py#L276-L278